option optimize_for = LITE_RUNTIME;

message PetUplevelRequest
{
 required uint32 bag_type = 1;
 required uint32 pos = 2;
 repeated uint32 item_id = 3;
 repeated uint32 item_num = 4;
}
